prody/atomic/functions.py

```diff
@@ -55,7 +55,7 @@ def saveAtoms(atoms, filename=None, **kwargs):
 
     if filename is None:
         filename = ag.getTitle().replace(' ', '_')
-    if '.ag.npz' not in filename:
+    if filename.endswith('.npz'):
```

Comment:
this should still be `'ag.npz'`

Reply:
I think we should just check for ".npz" otherwise if the original name is "xxx.npz", it would rename it to "xxx.npz.ag.npz". The current change is consistent with the behavior in `saveModel` where it only checks for ".npz". Ultimately, as long as it is an NPZ file, the secondary suffix shouldn't matter.

However, I just realized that in many places we didn't consider the case sensitivity. So I am making changes in all those places.